Tasks/MavenV3/task.json

```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 3,
         "Minor": 214,
-        "Patch": 0
+        "Patch": 1
```

Comment:
Could you describe the reason for bumping the version of this task?

Reply:
I made some changes in packaging-common directory, and it requires version bump for all the task that uses it